src/modelfree/shaping_wrappers.py

```diff
@@ -118,28 +141,35 @@ def predict(self, observation, state=None, mask=None, deterministic=False):
 
 
 def apply_reward_wrapper(single_env, shaping_params, agent_idx, scheduler):
-    anneal_frac = shaping_params.get('anneal_frac')
-    if anneal_frac is not None:
-        annealer = LinearAnnealer(1, 0, anneal_frac).get_value
-        scheduler.set_func('rew_shape', annealer)
-        rew_shape_func = scheduler.get_func('rew_shape')
+    if 'metric' in shaping_params:
```

Comment:
Guessing the type based on which keyword arguments are present is a little hacky, but it's OK for now as we only have two annealer types in use. But if we add more annealers would consider making it a more direct config format, where you specify the annealer type and then annealer-specific arguments.